docs/games/0002-StardewValley.md

```diff
@@ -0,0 +1,143 @@
+# Stardew Valley
+
+## Stores and Ids
+
+- [Steam](https://store.steampowered.com/app/413150/Stardew_Valley/): `413150`
+- [GOG](https://www.gog.com/game/stardew_valley): `1453375253`
+- [Xbox Game Pass](https://www.xbox.com/en-US/games/store/p/9MWR1NC6VQ6L): `UNKNOWN`
+
+## Engine and Mod Support
+
+Stardew Valley uses .NET 5 and [MonoGame](https://github.com/MonoGame/MonoGame), which is an open-source implementation of the discontinued [XNA Framework](https://en.wikipedia.org/wiki/Microsoft_XNA). MonoGame, and Stardew Valley to that extend, is fully cross-platform and runs natively on desktop (Windows/Linux/macOS), on the phone (Android/iOS) and on the console (PS4/PS5/Xbox using UWP and XDK/Nintendo Switch).
+
+Modding is done via [Stardew Modding API](https://smapi.io/), or _SMAPI_ for short. SMAPI is an [open-source](https://github.com/Pathoschild/SMAPI) modding framework and is supported on [Windows](https://stardewvalleywiki.com/Modding:Installing_SMAPI_on_Windows), [Linux](https://stardewvalleywiki.com/Modding:Installing_SMAPI_on_Linux) ([Steam Deck](https://stardewvalleywiki.com/Modding:Installing_SMAPI_on_Mac)), [macOS](https://stardewvalleywiki.com/Modding:Installing_SMAPI_on_Mac) and [Android](https://stardewvalleywiki.com/Modding:Installing_SMAPI_on_Android) (experimental at best).
+
+## Game Files
+
+- Windows: `Stardew Valley.exe`
+- Linux and macOS: `StardewValley` (shell script that launches `Stardew Valley`)
+
+The shell script only checks if the current macOS version is supported and launches the game.
+
+Saves are located inside the AppData folder (`%AppData%` on Windows, `~/.config` on Linux) under `StardewValley/Saves`. SMAPI puts the saves inside `StardewValley/.smapi` instead (this has changed in [`3.2`](https://github.com/Pathoschild/SMAPI/blob/9763bc7484e29cbc9e7f37c61121d794e6720e75/src/SMAPI.Installer/InteractiveInstaller.cs#L358), the previous location was `StardewValley/Saves/.smapi`).
```

Comment:
SMAPI does not move saves at all. (It does have a bundled mod that does save backups)